src/celmodel/model.cpp

```diff
@@ -8,10 +8,12 @@
 // as published by the Free Software Foundation; either version 2
 // of the License, or (at your option) any later version.
 
-#include "model.h"
+#include <algorithm>
 #include <cassert>
 #include <functional>
-#include <algorithm>
+
+#include <celutil/reshandle.h>
+#include "model.h"
 
 using namespace cmod;
```

Comment:
as you touched this file, then please replace this with
```
namespace cmod
{
```

Reply:
Namespace/includes cleanup done.